### PR TITLE
Component/info component

### LINKED ISSUE #12: info button
Info button to navigate to info component with information about current campaign

### LINKED ISSUE #13: info component
Info Component with information about current campaign

### PR DESCRIPTION
**Pull request by wleklinskimateusz**
closes #13 
closes #12 
## Changes:
- add colors to themes
- create useAppTheme hook with correct typing
- create InfoComponent

### CODE REVIEW

File: src/components/InfoComponent/InfoComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    shadowRadius: 4,
+    elevation: 5,
+  },
+  button: {
+    borderRadius: 20,
+    padding: 10,
+    margin: 10,
+    elevation: 2,
+  },
+  buttonInfo: {},
```
Comment (5ZYSZ3K) [inline src/components/InfoComponent/InfoComponent.tsx]: leftover

File: src/components/InfoComponent/InfoComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import {useAppTheme} from '../../hooks';
+import {InfoComponentProps} from './type';
+
+export const InfoComponent: FC<InfoComponentProps> = ({
+  campaignDescription,
+  campaignName,
+}) => {
+  const [modalVisible, setModalVisible] = useState(false);
+  const {colors} = useAppTheme();
+  return (
```
Comment (5ZYSZ3K) [inline src/components/InfoComponent/InfoComponent.tsx:15]: newline

File: src/components/InfoComponent/InfoComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    <View style={styles.centeredView}>
+      <Modal
+        animationType="slide"
+        transparent={true}
+        visible={modalVisible}
+        onRequestClose={() => setModalVisible(!modalVisible)}>
```
Comment (5ZYSZ3K) [inline src/components/InfoComponent/InfoComponent.tsx]:
```suggestion
        onRequestClose={() => setModalVisible((state) => !state)}>
```

File: src/components/InfoComponent/InfoComponent.tsx (hunk 1 above)
Comment (wleklinskimateusz) [inline src/components/InfoComponent/InfoComponent.tsx]:
```suggestion
```

File: src/components/InfoComponent/InfoComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+import {useAppTheme} from '../../hooks';
+import {InfoComponentProps} from './type';
+
+export const InfoComponent: FC<InfoComponentProps> = ({
+  campaignDescription,
+  campaignName,
+}) => {
+  const [modalVisible, setModalVisible] = useState(false);
+  const {colors} = useAppTheme();
+  return (
```
Comment (wleklinskimateusz) [inline src/components/InfoComponent/InfoComponent.tsx:15]:
```suggestion

  return (
```

**Pull request merged**